_data/library/scalalibs.yml

```
@@ -108,10 +108,10 @@
       url: https://github.com/chandu0101/sri
       desc: 'Sri (Scala React interface) is a scalajs library to build truly native cross platform (mobile (ios and android) and web) applications.'
       dep: '"com.github.chandu0101.sri" %%% "web" % "0.5.0"'
-    - name: OutWatch
+    - name: Outwatch
       url: https://outwatch.github.io/
-      desc: 'A functional & reactive UI framework based on Rx'
-      dep: '"io.github.outwatch" %%% "outwatch" % "0.7.1"'
+      desc: 'The Functional and Reactive Web-Frontend Library for ScalaJS'
```

Only a typo in the name of Scala.js. Otherwise LGTM.
```suggestion
      desc: 'The Functional and Reactive Web-Frontend Library for Scala.js'
```

---

Good point :+1:
Will update in other places as well!